docs/content/build-image.md

```diff
@@ -0,0 +1,358 @@
+ tar -xzf helm.tgz && \
+ mv linux-amd64/helm /usr/local/bin && \
+ rm helm.tgz
+RUN helm init --client-only
+```
+
+First, the `helm` mixin is copied instead of `exec` mixin. The Dockerfile looks similar in the beginning, but we can then see our next difference. The following lines of our generated Dockerfile were contributed by the `helm` mixin:
+
+```
+RUN apt-get update && \
+ apt-get install -y curl && \
+ curl -o helm.tgz https://storage.googleapis.com/kubernetes-helm/helm-v2.11.0-linux-amd64.tar.gz && \
+ tar -xzf helm.tgz && \
+ mv linux-amd64/helm /usr/local/bin && \
+ rm helm.tgz
+RUN helm init --client-only
+```
+
+How did that happen? To find out, let's first look at the `helm` mixin:
+
+```console
+~/.porter/mixins/helm/helm
+A helm mixin for porter 👩🏽‍✈️
+
+Usage:
+  helm [command]
+
+Available Commands:
+  build       Generate Dockerfile lines for the bundle invocation image
+  help        Help about any command
+  install     Execute the install functionality of this mixin
+  install     Execute the install functionality of this mixin
+  version     Print the mixin version
+
+Flags:
+      --debug   Enable debug logging
+  -h, --help    help for helm
+
+Use "helm [command] --help" for more information about a command.
+```
+
+The [Porter Mixin Contract](#tbd) specifies that mixins must provide a `build` sub command that generates Dockerfile lines to support the runtime execution of the mixin. In the case of the `helm` mixin, this includes installing Helm and running a `helm init --client-only` to prepare the image. At build time, Porter uses the _porter.yaml_ to determine what mixins are required for the bundle. Porter then invokes the build sub-command for each specified mixin and appends that output to the base Dockerfile.
+
+## Including The Dependencies
+
+Now that we've seen how Porter utilizes mixins to build an invocation image, it is time to address how dependencies are included. Consider the following sample `porter.yaml`:
+
+```yaml
+mixins:
+- exec 
+
+name: dependency-example 
+version: 0.1.0
+invocationImage: jeremyrickard/dependency-example:latest
+
+dependencies:
+- name: mysql
+  parameters:
+    database_name: wordpress
+    mysql_user: wordpress
+
+install:
+- description: "Say Hello"
+  exec:
+    command: bash
+    arguments:
+      - -c
+      - echo Hello World
+
+uninstall:
+- description: "Say Goodbye"
+  exec:
+    command: bash
+    arguments:
+      - -c
+      - echo Goodbye World
+```
+
+This bundle, for example, declares a dependency on a bundle named `mysql`. The CNAB specification doesn't provide a mechanism for handling dependency resolution. Porter addresses this capability by resolving any dependencies at build time, including the contents of each dependency in the invocation image. At runtime the contents of that bundle will therefore be in the bundle and the Porter runtime component can execute them successfully.
```

Comment:
Change `Port addresses this capability` to `gap`

Reply:
Or to be more positive, `Porter supplements the CNAB spec to support dependencies by resolving ....`